tapir/coop/templates/coop/welcome_desk_share_owner.html

```diff
@@ -22,33 +22,41 @@ <h5 class="card-header">
         <div class="card-body">
             {% if can_shop %}
                 <div class="alert alert-success" role="alert" id="welcome_desk_can_shop">
-                    {% blocktranslate with name=share_owner.get_info.first_name %}{{ name }}  can shop :){% endblocktranslate %}
+                    {% blocktranslate with name=share_owner.get_info.first_name %}{{ name }}  can shop
```

Comment:
Was this an auto-formatter? It would be better to have the text content on one line. Same for the rest of the file

Reply:
Yeah the default autoformatter doesn't give nice results for the template files. I still like formatting them because we get results independent from the author.
Do we want to increase the max line length for HTML files and un-ignore .idea/codeStyles/Project.xml? It specific for PyCharm could still be a reference.